### PR TITLE
Add fast_integration environment

### PR DESCRIPTION
**Pull request by dincho**
in scope of https://www.pivotaltracker.com/story/show/154634483

depend on https://github.com/aeternity/infrastructure/pull/33 (fast_integration is deployed manually already, so this can be merged)

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -602,3 +608,23 @@ workflows:
...
+      - schedule:
+          cron: "0 23 * * *"
+          filters:
+            branches:
+              only:
+                - master
+    jobs:
+      - deploy_fast_integration:
+          requires:
+            - test
```
Comment (lucafavatella) [inline .circleci/config.yml]: Do these jobs get triggered automatically overnight because of this dependency?

Comment (dincho) [reply .circleci/config.yml]: Good question. I'll shall setup a mock

Comment (dincho) [reply .circleci/config.yml]: Theoretically, `test`, `eunit`, `static_analysis` and `linux_package` run on all branches and `uat_tests` run on `master`, so it looks like that should be fine.

Comment (dincho) [reply .circleci/config.yml]: I pushed a mock in 40145eb1 that shall run at 9:30 UTC (after ~10mins)

Comment (dincho) [reply .circleci/config.yml]:
```
We weren't able to start this workflow.
The following jobs are unreachable: deploy_fast_integration
At least one job in the workflow must have no dependencies.
deploy_fast_integration: requires job "static_analysis" but "static_analysis" is not part of this workflow.
deploy_fast_integration: requires job "eunit" but "eunit" is not part of this workflow.
deploy_fast_integration: requires job "linux_package" but "linux_package" is not part of this workflow.
deploy_fast_integration: requires job "test" but "test" is not part of this workflow.
```

Comment (dincho) [reply .circleci/config.yml]: @lucafavatella I pushed new mock 9be03d24

I guess we should just assume the master is in good shape and depend only on `linux_package`

Comment (dincho) [reply .circleci/config.yml]: https://circleci.com/workflow-run/7b17470a-4373-4f19-b68c-797cf02fde98

Comment (dincho) [reply .circleci/config.yml]: Passed. I prepared changes in 7e592f33.

I prefer to "risk" and rely on stable master than copy pasting all the jobs in this cron workflow.

File: .circleci/config.yml [hunk 2]
```diff
@@ -602,3 +608,19 @@ workflows:
...
+      - schedule:
+          cron: "0 23 * * *"
+          filters:
+            branches:
+              only:
+                - master
+    jobs:
+      - deploy_fast_integration:
+          requires:
+            - linux_package
```
[the remark this answers is not in the document]
Comment (dincho) [reply .circleci/config.yml]: I thought I added it, at least it was in the mock/test workflow. I'll fix that in a while.

[the remark this answers is not in the document]
Comment (dincho) [reply .circleci/config.yml]: @lucafavatella fixed in 9a827ad9

**Pull request merged**